### PR TITLE
remove output before db test, tweaks to bufr2ioda test

### PR DESCRIPTION
**Pull request by AndrewEichmann-NOAA**
pre-cleans soca obsdb test, plus some refinement of bufr2ioda ctest

### CODE REVIEW

File: test/soca/test_bufr2ioda.sh [hunk 1]
```diff
@@ -4,13 +4,21 @@ set -x
...
+rm -v ${PROJECT_BINARY_DIR}/test/${OUTFILE}
+
 ${CMD} ${OBSYAML}
+rc=$?
 
+ls ${PROJECT_BINARY_DIR}/test/${OUTFILE}
+ra=$?
+rc=$((rc+ra))
 
+export err=$rc; err_chk
```
Comment (guillaumevernieres) [inline test/soca/test_bufr2ioda.sh]: `err_chk` is not defined. I don't know where this is coming from, global-workflow? I think It's not needed here.

Comment (AndrewEichmann-NOAA) [reply test/soca/test_bufr2ioda.sh]: It's not needed, but oddly enough it does work. I never did figure out where it came from.

**Comment by guillaumevernieres** [comment]
looks like your prep and run step failed @AndrewEichmann-NOAA

**Comment by AndrewEichmann-NOAA** [comment]
@guillaumevernieres Now I got it - I accidentally changed the DB root directory, which made the automatic tests fail, my build was still using the previously created DB in the correct directory, which is why prep and run worked for me. Example of the evil I'm trying to correct

**Review by guillaumevernieres — APPROVED** [review]
:+1:

**Pull request merged**